.changeset/five-waves-relax.md

```diff
@@ -0,0 +1,9 @@
+---
+"stylelint": patch
+---
+
+Fixed: `annotation-no-unknown` false positives in CSS-in-JS syntax
+Fixed: `function-name-case` false positives in CSS-in-JS syntax
+Fixed: `function-no-unknown` false positives in CSS-in-JS syntax
+Fixed: `unit-no-unknown` false positives in CSS-in-JS syntax
+Fixed: `value-keyword-case` false positives in CSS-in-JS syntax
```

Comment:
```suggestion
Fixed: `*-case|*-unknown` false positives for CSS-in-JS template literals
```

Reply:
I think this would be confusing to users. I would understand proposed line as if _all_ rules matching the pattern would have fixed false positives. I fixed only rules that I personally had issues with. I don't know about other rules.

Reply:
> I fixed only rules that I personally had issues with. I don't know about other rules.

Oh, I didn't clock that. Makes sense. I've updated the suggestion.